modules/compatibility-tests/src/jobs/java/org/apache/ignite/internal/compute/SendAllMetastorageCommandTypesJob.java

```diff
@@ -58,7 +58,7 @@ public CompletableFuture<Void> executeAsync(JobExecutionContext context, String
                             iif(exists(ByteArray.fromString("key")), ops().yield(), ops().yield())),
                     metastorage.evictIdempotentCommandsCache(HybridTimestamp.MAX_VALUE),
                     sendCompactionCommand(metastorage)
-            );
+            ).thenCompose((v) -> metastorage.storage().flush());
```

Comment:
Maybe we should add a TODO with an automatic check to ensure all write commands are tested? Like `org.apache.ignite.internal.RaftCommandsUnitCompatibilityArchTest`, for example.